Dockerfile

```diff
@@ -33,63 +31,49 @@ RUN apt install -y --no-install-recommends openmpi-bin openssh-server openssh-cl
 RUN mkdir -p /deps
 WORKDIR /deps
 
+RUN mkdir -p /mfu
 RUN wget https://github.com/hpc/libcircle/releases/download/v0.3/libcircle-0.3.0.tar.gz \
     && tar xfz libcircle-0.3.0.tar.gz \
     && cd libcircle-0.3.0 \
-    && ./configure --prefix=/usr/lib \
+    && ./configure --prefix=/deps/libcircle/lib \
+    && make install
+
+RUN wget https://github.com/libarchive/libarchive/releases/download/v3.5.3/libarchive-3.5.3.tar.gz \
+    && tar xfz libarchive-3.5.3.tar.gz \
+    && cd libarchive-3.5.3 \
+    && ./configure --prefix=/deps/libarchive/lib \
     && make install
 
 RUN wget https://github.com/llnl/lwgrp/releases/download/v1.0.3/lwgrp-1.0.3.tar.gz \
     && tar xfz lwgrp-1.0.3.tar.gz \
     && cd lwgrp-1.0.3 \
-    && ./configure --prefix=/usr/lib \
+    && ./configure --prefix=/deps/lwgrp/lib \
     && make install
 
 RUN wget https://github.com/llnl/dtcmp/releases/download/v1.1.1/dtcmp-1.1.1.tar.gz \
     && tar xfz dtcmp-1.1.1.tar.gz \
     && cd dtcmp-1.1.1 \
-    && ./configure --prefix=/usr/lib --with-lwgrp=/usr/lib \
-    && make install
-
-RUN wget https://github.com/libarchive/libarchive/releases/download/v3.5.3/libarchive-3.5.3.tar.gz \
-    && tar xfz libarchive-3.5.3.tar.gz \
-    && cd libarchive-3.5.3 \
-    && ./configure --prefix=/usr/lib \
+    && ./configure --prefix=/deps/dtcmp/lib --with-lwgrp=/deps/lwgrp/lib \
     && make install
 
+RUN mkdir -p /mfu
 ARG MPI_FILE_UTILS_VERSION="0.11"
```

Comment:
Not sure how much this version changes, but I'd like to see this after the FROM line especially if we use a specific version for the image. Mostly just for clarity.

Reply:
This doesn't relate to the image. It's a variable for the next line. For locality, I'd like to remain where it currently is.